apps/mosaico/src/Article/Article.purs

```diff
@@ -84,6 +87,7 @@ render imageComponent props =
         bodyWithoutAd = map (renderElement (Just props.paper) imageComponent (Just props.onArticleClick)) body
         bodyWithAd = map (renderElement (Just props.paper) imageComponent (Just props.onArticleClick))
           <<< insertAdsIntoBodyText "mosaico-ad__bigbox1" "mosaico-ad__bigbox2" $ body
+        advertorial = fromMaybe (DOM.div_ []) (renderAdvertorialTeaser <$> props.advertorial)
```

Comment:
As `JSX` is a `Monoid`, we could use `foldMap` here too:
```
advertorial = foldMap renderAdvertorialTeaser props.advertorial
```

Reply:
Thanks, fixed.